Source/ParameterEncoder.swift

```diff
@@ -221,6 +224,45 @@ public final class URLEncodedFormEncoder {
         }
     }
 
+    /// Configures how `Date` parameters are encoded.
+    public enum DateEncoding {
+        /// Encodes the date from `Date` structure.
+        case deferredToDate
+        /// Encodes the date as a UNIX timestamp.
+        case secondsSince1970
+        /// Encodes the date as UNIX millisecond timestamp.
+        case millisecondsSince1970
+        /// Encodes the date as an ISO-8601-formatted string (in RFC 3339 format).
```

Comment:
Probably clearer to say: "Encodes dates according to the ISO 8601 and RFC3339 standards.", similar to the `JSONEncoder` documentation.

Reply:
Done.